### DISCUSSION TITLE
Not able to run ollama server command

### DISCUSSION THREAD

**Discussion by wghaight** [logancyang/obsidian-copilot#554] [Q&A]
'OLLAMA_ORIGINS' is not recognized as an internal or external command, operable program or batch file.

This is the error I get upon running the serve command.

I have tried changing dir to windows\system32

Still get same error.

Any help will be appreciated!

Thank you, Walter

**Comment by jritsema** [comment]
It looks like you may be trying to run mac/linux command on windows, which won't work.  See https://github.com/logancyang/obsidian-copilot/blob/master/local_copilot.md#ollama-for-windows-preview
